TeslaMateAgile/appsettings.json

```
@@ -18,5 +18,8 @@
   "Awattar": {
     "BaseUrl": "https://api.awattar.de/v1",
     "VATMultiplier": 1.19
-  }
+  },
+  "Barry": {
+    "BaseUrl": "https://jsonrpc.barry.energy/json-rpc",
```

Last item in array should not have comma on it

```suggestion
    "BaseUrl": "https://jsonrpc.barry.energy/json-rpc"
```